applications/e-commerce/Sample.html

```diff
@@ -0,0 +1,46 @@
+<head>
+
+    <body>
+        <header>
+            <h2>
+                Welcome to E-commerce website
+            </h2>
+        </header>
+
+        <main>
+            <section>
+                <h4>What is E-commerce Wbsite?</h4> 
+                <p> <b>Ecommerce or electronic commerce</b> refers to a business model that 
+                    involves sales transactions being done on the <strong>web</strong>. 
+                    Virtually every online shopping website - big or small - follows this structure. 
+                    Any site where you can obtain items for sale over the internet 
+                    is considered an <em> ecommerce website.</em></p>
+            </section>
+            <hr />
+
+            <aside>
+                <h4>Types of transiction in E-commerce</h4>
+                <ol>
+                    <li>B2B(Business to Business)</li>
+                    <li>B2C(Business to Customer)</li>
+                    <li>C2C(Customer to Customer)</li>
+                    <li>C2B(Customer to Business)</li>
+                </ol>
+            </aside>
+
+            <h4>Examples of E-commerce</h4>
+            <ul>
+                <li>Amazon</li>
+                <li>Wish</li>
+                <li>Pc Optimum</li>
+                <li>And So on...</li>                
+            </ul>
+        </main>
+
+        <footer>
+            Contact, Map/Location, other links...
+        </footer>
+        
+
+    </body>
+</head>
```

Comment:
Your content shouldn't go in the `<head>` element.

Reply:
oohhh... I think I should go through before submit it.... :D 
Thank you Mark...